### DISCUSSION TITLE
Is there a way to start rnote from the command line specifying rnote file path?

### DISCUSSION THREAD

**Discussion by kamoii** [flxzt/rnote#921] [Q&A]
For example, something like:

    $ rnote a/b/foo.rnote

From the output of `rntoe --help-all`, it seems impossible, but maybe its not just documented?

**Comment by flxzt** [comment]
Opening a file from a specified path with: `rnote some/path/doc.rnote` works for me. What does your cli output say when you are executing it?

**Reply by kamoii** [reply]
I'm sorry. It was failing because I mistakenly specified a non-existent rnote file path. When I specified an existing rnote file path, I was able to open it without any problem.

**Reply by flxzt** [reply]
No problem
